### ISSUE TITLE
can not tar -xvf the pretrained model.

### ISSUE THREAD

**Issue by DefTruth** [louis-she/center-loss.pytorch#13]
can not tar -xvf the pretrained model.
